### PR TITLE
[session3] 김현서 과제 제출합니다.

### PR DESCRIPTION
**Pull request by neulbokim**
통계 과제와 회귀기초 사전과제를 진행했습니다.

이번 과제를 진행하면서 인상 깊었던 점, 느낀 점은 다음과 같습니다.

### 통계 과제
* 귀무가설 설정 방법: 보통 "~~의 관련이 없다" 등으로 서술됨 (내가 주장하고자 하는 바와 상관없이)
* 범주형-범주형 시각화하고 해석하는 게 좀 어려운 것 같음 (특히 모자이크 플롯: 내가 그린 플롯은 그 의미가 한눈에 잘 안 들어오는 것 같다. 'Label Encoding'으로 바꿔서 봐볼까? 했는데 Label Encoding은 숫자를 자동으로 매칭해주는 것 같아서 적합하지 않은 것 같았음. 내가 하고 싶은 건 구글 플레이스토어 별점처럼 (0점: 매우 안 좋음, 5점: 매우 좋음) 나타내는 거였는데.. 순서가 있게 매칭하는 걸 어떻게 할지(0점: Fair, 5점: Ideal 같이) 딱히 안 떠올랐음(딕셔너리로 하면 되려나?). 근데 통기스 1차시에서 구글 플레이스토어 별점 같은 경우 명확히 어떤 자료형이라고 말하기 어렵다고 했는데 이건 좀 더 생각해봐야 될 것 같음.) 암튼 범주형-범주형을 좀 더 공부해봐야겠음.

### 회귀기초 사전과제
* 인공지능개론(딥러닝개론)에서 배웠을 때는 train set, validation set, test set 이 있었던 것 같은데, 이거 딥러닝에만 있던 거라 했었나? 잘 기억이 안난다. 머신러닝에서는 train set이랑 test set만 주로 있고 딥러닝에서는 train set, validation set, test set 이렇게 있다고 배웠던 것 같기도. 이것도 찾아봐야겠다.
* 다중공선성: 이름은 너무 어려워보이는데 나름 친숙한 개념이었다. 중고등학교 때에도 배웠던.. 인과관계와 상관관계를 혼동하면 안 된다는 것의 연장선인 듯? 
* VIF지수, 규제 선형 모델(릿지, 라쏘, 엘라스틱넷) 개념은 처음 봤다... 세션 열심히 들어야지
* PCA 개념도 처음 봤는데 꽤 이해가 잘 됐음
* t검정 개념 익숙하지 않으니까 세션 열심히 들어야겠다(특히 수식)

역시 날이 갈수록 점점 내용이 어려워지고 있는 것 같아요 💦😅 계속 더더 열심히 해야겠습니다!! 파이팅!!!

### CODE REVIEW

**Comment by hyunji1122** [comment]
제가 본 마크다운 중에 제일 잘 정리되어있고 깔끔한 요약인 것 같습니다ㅎㅎ 
이대로 교육자료에 써도 되겠어요!!!!

이번 과제도 너무너무 수고하셨고, 과제 모두 잘 해주셨습니다!!
버디조 톡방에서의 토론도 인상깊었어요🤩

이번에 배울 회귀 내용이 양도 많고 중요하니까, 늘 그래오셨던 것 처럼 집중하셔서 교육세션 들으시면 좋을 것 같습니다!!!
